packages/constants/package.json

```diff
@@ -2,9 +2,23 @@
   "name": "@plane/constants",
   "version": "0.26.1",
   "private": true,
-  "main": "./src/index.ts",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/**"
+  ],
   "license": "AGPL-3.0",
+  "scripts": {
+    "build": "tsup ./src/index.ts --format esm,cjs --dts --minify"
+  },
```

Comment:
@sriramveeraghanta I added this due to the above as it seems to need a build step